sycl/doc/extensions/supported/sycl_ext_oneapi_bfloat16.asciidoc

```diff
@@ -41,18 +41,7 @@ https://github.com/intel/llvm/issues
 
 This extension is written against the SYCL 2020 specification, Revision 5.
 
-== Status
 
-This extension is implemented and fully supported by DPC++, with the
-limitation noted below.
-
-[NOTE]
-This extension has limited support in ahead-of-time (AOT) compilation mode when
-the target device is an Intel GPU.  Software emulation of this feature does not
-work in this case, so AOT mode is only supported if the target device has
-native support for `bfloat16`. Currently, this is limited to Intel Xe HP GPUs.
-The DPC++ implementation has no such limitation in just-in-time (JIT)
-compilation mode.
 
```

Comment:
Did you intend to remove the entire "Status" section?

Reply:
Yes, because it was added for mentioning the AOT limitation, which has been removed.
Is there a need for a Status section and what should be put there?

Reply:
Yes, there should always be a Status section.  If the extension is supported without limitations it should say:

> This extension is implemented and fully supported by DPC++.

See the [template](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/template.asciidoc#status).

Reply:
Ok, I've added a Status section.